pkg/applicationserver/io/packages/loradms/v1/package.go

```diff
@@ -334,7 +336,7 @@ func float64PtrOfTimestamp(x *pbtypes.Timestamp) *float64 {
 	if x == nil {
 		return nil
 	}
-	f := float64(ttnpb.StdTime(x).Unix())
+	f := float64(ttnpb.StdTime(x).UnixNano())
```

Comment:
This is not correct. What I expected was `float64(ttnpb.StdTime(x).UnixNano()) / float64(1e9)` - the unit has to be seconds, not nano seconds.